src/modules/chat_commands/store.js

```diff
@@ -0,0 +1,37 @@
+import twitch from '../../utils/twitch.js';
+import watcher from '../../watcher.js';
+
+export const PermissionLevels = {
+  VIEWER: 0,
+  VIP: 1,
+  MODERATOR: 2,
+  BROADCASTER: 3,
+};
+
+class CommandStore {
+  constructor() {
+    this.commands = [];
+    watcher.on('load.chat', () => this.loadCommands());
+  }
+
+  loadCommands() {
+    const twitchCommandStore = twitch.getChatCommandStore();
+    if (twitchCommandStore == null || this.commands.length === 0) {
+      return;
+    }
+    this.commands.forEach((command) => {
+      twitchCommandStore.addCommand(command);
+    });
+  }
+
+  registerCommand(command) {
+    const twitchCommandStore = twitch.getChatCommandStore();
+    if (twitchCommandStore == null) {
```

Comment:
this looks like a bug. commands will be unable to be registered when chat is not loaded. you want to always load them into the commands list, and insert them into the command store if that is available (otherwise it will happen later).